### PR TITLE
global: deploy to prod images

### PR DESCRIPTION
**Pull request by drjova**

### CODE REVIEW

**Review by michamos — APPROVED** [review]

**Review by michamos — CHANGES_REQUESTED** [review]
not using travis but github actions here

**Comment by drjova** [comment]
🤦

**Comment by drjova** [comment]
@michamos updated

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -65,3 +65,14 @@ jobs:
...
+        uses: peter-evans/repository-dispatch@v1
+        with:
+          repository: inspirehep/kubernetes
+          token: ${{ secrets.INSPIRE_BOT_TOKEN }}
+          event-type: deploy
+          client-payload: >
+            {"environment": "prod",
+             "image": "inspirehep/editor",
+             "tag": "${{ steps.describe.outputs.tag }}"}
+        if: ${{ github.event_name == 'release' }}
```
Comment (michamos) [inline .github/workflows/ci.yml:78]: are you sure this will work? it doesn't look like we'll build the image in this case

Comment (drjova) [reply .github/workflows/ci.yml:79]: you're right the tagged image will not be there, not sure about the syntax though

**Review by michamos — APPROVED** [review]

**Pull request merged**